test/Evolve.IntegrationTest.Cassandra/MigrationTest.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Cassandra.Data;
+using Evolve.Test.Utilities;
+using System.Collections.Generic;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Evolve.IntegrationTest.Cassandra
+{
+    [Collection("Database collection")]
+    public class MigrationTest
+    {
+        private readonly CassandraFixture _cassandraFixture;
+        private readonly ITestOutputHelper _output;
+
+        public MigrationTest(CassandraFixture cassandraFixture, ITestOutputHelper output)
+        {
+            _cassandraFixture = cassandraFixture;
+            _output = output;
+            if (!TestContext.Travis && !TestContext.AppVeyor)
+            { // AppVeyor and Windows 2016 does not support linux docker images
+                cassandraFixture.Start(fromScratch: true);
+            }
+        }
+
+        [SkipOnAppVeyorFact(DisplayName = "Run_all_Cassandra_migrations_work")]
+        public void Run_all_Cassandra_migrations_work()
```

Comment:
Why not complete this test and make it looks like the others from others databases ?

Reply:
I don't understand :-)

What is missing?

Reply:
Look at the [SqlServerMigrationTest](https://github.com/lecaillon/Evolve/blob/master/test/Evolve.IntegrationTest.SQLServer/MigrationTest.cs) for example.
I test all the Evolve commands.

Reply:
I'll try the copy/paste and adaptation.

Reply:
@lecaillon I'm running into issues with the ```Erase()``` method. It tries to run this in a transaction, which is again posing problems with Cassandra's drivers.

I'm thinking on how to fix this (maybe by adding a Cassandra property, just like ```SQLiteInMemoryDatabase```; this would be a quick and dirty fix (and private).

In the long run, I'm wondering if ```WrappedConnection``` shouldn't become abstract and have multiple implementations to alleviate these kinds of issues in a cleaner way.